dist/session.js

```diff
@@ -157,7 +160,7 @@ session.delimiter = function (str) {
   if (str === undefined) {
     return this._delimiter;
   }
-  this._delimiter = String(str).trim() + ' ';
```

Comment:
you bet. the PR is here anyways. let me know how I can help.

I did everything I could for now. I believe the ball is in your court now.

Let me know when you revert the changes so that I can re-open the issue with the ability to run prompts inside the parse context.